features/polyExplorer/src/components/embeddedSankey/embeddedSankey.jsx

```diff
@@ -105,17 +83,39 @@ const EmbeddedSankey = ({ links, groups }) => {
             />
             <p>{groups.source.label}</p>
             <Chips
-                chipsContent={displayedSourceChips}
-                activeChips={activeSources.length > 1 ? "all" : activeSources}
+                chipsContent={sources}
+                defaultActiveChips={
+                    activeSources.length > 1 ? ["all"] : activeSources
+                }
                 onChipClick={handleActiveSourceChange}
+                exclusive={false}
+                all={groups.source.all ? { translation: "All" } : null}
+                others={
+                    groups.source.others
+                        ? {
+                              translation: "others",
+                              ids: groups.source.others,
+                              exclusive: false,
+                          }
+                        : null
+                }
             />
             <p>{groups.target.label}</p>
             <Chips
-                chipsContent={displayedTargetChips}
-                activeChips={
-                    activeTargets.length > 1 ? "others" : activeTargets
-                }
+                chipsContent={targets}
+                defaultActiveChips={[...targets, "others"]}
                 onChipClick={handleActiveTargetChange}
+                exclusive={false}
+                all={groups.target.all ? { translation: "All" } : null}
```

Comment:
Would it be possible to add `all` and `others` as part of `source` and `targets`, with the embedded logic and differences? For instance, right now all and others can be active or inactive, but that does not apply to them, right?

Reply:
The problem is that the all and others chip work totally different than the rest. Before the behaviour was exactly that, i would just add them as chips in the parent and regard their cases there. Now they have special behaviour: AllChip will deactivate the other chips or when another chip is clicked all will be deactivated. Other will encapsulate different chips which it needs to delete as chips but still trigger them indirectly and so ... This is why this got so messy

Reply:
Right, but think about some additional behavior we might want in a chip set. By having that behavior encoded in the chipset, you would need to add more "if"s there. By encapsulating the behavior in the chip itself, it might be easier to just add new toggle chips or whatever without needing to add more code in Chips to accomodate them.
Anyway, it's a good solution for now, and possibly outside the scope of this PR; we might want to think about it in the future, maybe create a Jira ticket for that.

Reply:
Ok I get it now, like in Tabs/Tab you would want the chips to know their own behaviour. Yeah thought about that as well, is a good refactoring for another day